node/ChainwebNode.hs

```diff
@@ -443,6 +482,11 @@ pkgInfoScopes =
 -- -------------------------------------------------------------------------- --
 -- main
 
+-- KILLSWITCH 2020-01-15T00:00:00Z for version 1.1
+--
+killSwitchDate :: Maybe String
+killSwitchDate = Just "2020-01-15T00:00:00Z"
```

Comment:
Should this not be a function of `ChainwebVersion`?

Reply:
I don't think so. It's a property of the cabal version.

Strictly speaking the chainwebVersion is `mainnet01` and this doesn't change when cabal version 1.1 goes offline.

Chainweb Version `mainnet01` is immutable, so everything we add to it has to stay. But a kill switch supposed to be removed while the chainweb version stays the same. 

So, `mainnet01` has a property that cabal version 1.1 isn't aware of yet. So cabal version 1.1 must go away before the respective property of `mainnet01` becomes relevant.

Thinking of the chainweb version as something immutable, and thinking of this change as property of the chainweb version that always existed (and always will exist) but wasn't yet represented in the code, makes it easier to reasoning about the correctness of the validation logic.